cmd/shared.go

```diff
@@ -32,3 +45,10 @@ func validateMaxArgs(args []string, max int) error {
 	}
 	return nil
 }
+
+func validateBooleanArgument(arg string) error {
```

Comment:
I liked validate as that matches the rest of the functions in this file. Currently `Ensure` is used elsewhere for exiting with error whereas the validate functions return an error.